test/e2e/framework/seclist_util.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"time"
+
+	client "github.com/oracle/oci-cloud-controller-manager/pkg/oci/client"
+	core "github.com/oracle/oci-go-sdk/core"
+)
+
+// CountSecListTunnelSinglePortRules counts the number of 'single port'
+// (non-ranged) egress/ingress rules for the specified list and port.
+func CountSecListTunnelSinglePortRules(oci client.Interface, egressSecListID, ingressSecListID string, port int) (int, int) {
```

Comment:
Thought this might come up! Just wanted to mention I preferred environment variables here as I only intended these tests to be executed by our CCM ci pipeline (where we manage the seclist ids). Not by the canary tests etc where we don't. I felt threading them through was a bit noisy. Still if you want flags for consistency ill do it.